### PR TITLE
Add news harsh demo code

### PR DESCRIPTION
**Pull request by SunnyStardust**
1. python testConn.py -add text.txt
Add the incident read from text.txt to the database
2. python testConn.py -get
Get all incidents from the database

[.txt](https://github.com/1thing-org/1thing-crawler/files/7921775/default.txt)

Change the name of "default.txt" to ".env" to run the service.

### CODE REVIEW

**Review by qz267 — CHANGES_REQUESTED** [review]
All LGTM, would you mind add some documentation to README.md to mention how to use the code?

**Comment by SunnyStardust** [comment]
> All LGTM, would you mind add some documentation to README.md to mention how to use the code?

README.md updated

File: README.md [hunk 1]
```diff
@@ -2,3 +2,8 @@
 Crawlers for 1thing project 
 
 [Demo slide](https://go.zheng.ai/1t/sc)
+
+# News Hash Service
+cd news_hash
+1. Add the incident read from text.txt to the database: python testConn.py -add text.txt
+3. Get all incidents from the database: python testConn.py -get
```
Comment (qz267) [inline README.md:9]: missing item `2`?

**Review by qz267 — CHANGES_REQUESTED** [review]
Missing item `2`?

**Review by qz267 — APPROVED** [review]

**Pull request merged**